Rakefile

```diff
@@ -4,6 +4,26 @@ require 'date'
 require 'json'
 require 'net/http'
 require 'optparse'
+require "rake/testtask"
+
+# To run all tests:
+#   rake (or) rake test
+#
+# To run one test file:
+#   ruby -Ilib:test test/FILENAME.rb
+#   (or)
+#   rake test TEST=test/FILENAME.rb
+#
+# To run an individual test in a test file:
+#   ruby -Ilib:test test/FILENAME.rb -n TEST_NAME
+#   (or)
+#   rake test TEST=test/FILENAME.rb TESTOPTS=--name=TEST_NAME
+#
+desc 'Run all tests with `rake` or `rake test`'
+task default: :test # Make test runner the default rake task.
+Rake::TestTask.new do |task|
+  task.pattern = 'test/test_*.rb'
```

Comment:
I don't understand ruby. Does this not cause `test_helper.rb` to run?

Reply:
Good eye! Yes, if the test files are not specified, e.g. `rake test` works, but `rake TEST=test/test_rake_posts_new.rb` will fail with `uninitialized constant Minitest` if the test file does not require `test_helper`.

FWIW we can name test files as '*_test.rb' if you prefer. In this case the test helper is never auto-run.